types/lua-livr/LIVR.d.tl

```diff
@@ -0,0 +1,38 @@
+--
+-- https://luarocks.org/modules/fperrad/lua-livr
+--
+
+local record LIVR
+   type Data = {string:any}
+   type Error = string|table
+   type Validator = function(value: any, all: Data): any, Error
+   type Builder = function(builders: {string:Builder}, ...: any): Validator
+   type Rule = string|table
+
+   record Alias
+      name: string
+      rules: Rule
+      error: string
+   end
+
+   record VALIDATOR
```

Comment:
Teal doesn't have hard conventions, so just take this as a mild suggestion:  generally all-upercase names are used for constants, so in a situation like this I'd use `Validator` for the record and `ValidatorFn` for the function-type from line 8 (I personally often use `FooFn` for function types).

 If you don't like this, feel free to mark this as resolved and I'll merge it regardless!